Remove hacky wrapper for about 100 kernels

wenleix opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#54367 Remove hacky wrapper for about 100 kernels**

Codemod commands generated by https://github.com/pytorch/pytorch/pull/54098

Differential Revision: [D27210057](https://our.internmc.facebook.com/intern/diff/D27210057/)

Commit:
Remove hacky wrapper for about 100 kernels

Codemod commands generated by https://github.com/pytorch/pytorch/pull/54098

Differential Revision: [D27210057](https://our.internmc.facebook.com/intern/diff/D27210057/)

[ghstack-poisoned]

Commit:
rebase & trailing space on "Remove hacky wrapper for about 100 kernels"


Codemod commands generated by https://github.com/pytorch/pytorch/pull/54098

Differential Revision: [D27210057](https://our.internmc.facebook.com/intern/diff/D27210057/)

[ghstack-poisoned]

Commit:
rebase on "Remove hacky wrapper for about 100 kernels"


Codemod commands generated by https://github.com/pytorch/pytorch/pull/54098

Differential Revision: [D27210057](https://our.internmc.facebook.com/intern/diff/D27210057/)

[ghstack-poisoned]

smessmer reviewed (approved):
Cool :)
Btw, how did you ensure that you got all at::native calls? In an op `my_op(const Tensor& self, Tensor &out)`, the compiler might not fail even if you forgot to swap the arguments at a call site.

wenleix commented:
@smessmer : 

> In an op my_op(const Tensor& self, Tensor &out), the compiler might not fail even if you forgot to swap the arguments at a call site.

It will fail because the wrong calling ordering will discard const qualification right?

Commit:
rebase on "Remove hacky wrapper for about 100 kernels"


Codemod commands generated by https://github.com/pytorch/pytorch/pull/54098

Differential Revision: [D27210057](https://our.internmc.facebook.com/intern/diff/D27210057/)

[ghstack-poisoned]

Commit:
rebase on "Remove hacky wrapper for about 100 kernels"


Codemod commands generated by https://github.com/pytorch/pytorch/pull/54098

Differential Revision: [D27210057](https://our.internmc.facebook.com/intern/diff/D27210057/)

[ghstack-poisoned]

Commit:
a few minor fixes on callsite on "Remove hacky wrapper for about 100 kernels"


Codemod commands generated by https://github.com/pytorch/pytorch/pull/54098

Differential Revision: [D27210057](https://our.internmc.facebook.com/intern/diff/D27210057/)

[ghstack-poisoned]
